lib/charms/mongodb/v0/mongo.py

```diff
@@ -270,6 +270,11 @@ def get_users(self) -> Set[str]:
             ]
         )
 
+    def get_all_users(self) -> Set[str]:
+        """Get all users, including the three charmed managed users."""
+        users_info = self.client.admin.command("usersInfo")
+        return {user_obj["user"] for user_obj in users_info["users"]}
```

Comment:
I need to check but this function is essentially here to get users to update/remove the charm users (`operator`, `backup` and `monitor`) to see which ones were created already or not.
This is here to make it more reliable in case some users were removed/not yet re-introduced in a storage.